sdk/core/azure-core/azure/core/rest/_helpers.py

```diff
@@ -68,7 +68,16 @@
 FileContent = Union[str, bytes, IO[str], IO[bytes]]
 FileType = Tuple[Optional[str], FileContent]
 
-FilesType = Union[Mapping[str, FileType], Sequence[Tuple[str, FileType]]]
+FilesType = Union[
+    # file (or bytes)
+    FileContent,
+    # (filename, file (or bytes))
+    Tuple[Optional[str], FileContent],
+    # (filename, file (or bytes), content_type)
+    Tuple[Optional[str], FileContent, Optional[str]],
+    # (filename, file (or bytes), content_type, headers)
+    Tuple[Optional[str], FileContent, Optional[str], Mapping[str, str]],
```

Comment:
LEt's remove additionanl heaers for now

Reply:
oh yes good point, didn't remove it from the types